### PR TITLE
Oncoprint Toolbar Improvements

### PR DESCRIPTION
**Pull request by Luke-Sikina**
Fix cBioPortal/cbioportal#6880

Describe changes proposed in this pull request:
- added a count bubble for Clinical Tracks and Heatmaps

# Any screenshots or GIFs?
Screenshot from 2019-12-06 11-36-21

### CODE REVIEW

File: src/pages/resultsView/oncoprint/OncoprintDropdownCount.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import * as React from "react";
+import {observer} from "mobx-react";
+
+@observer
+export default class OncoprintDropdownCount extends React.Component<{count: number | undefined}, {}> {
+    render() {
+        if (this.props.count === undefined) {
+            return <span/>
```
Comment (alisman) [inline src/pages/resultsView/oncoprint/OncoprintDropdownCount.tsx]: i think you can just return null, no?

**Comment by alisman** [comment]
@Luke-Sikina one more thing i just noticed: we want to change the button text from "Heatmaps" to "Add Heatmap Tracks"

**Comment by Luke-Sikina** [comment]
> @Luke-Sikina one more thing i just noticed: we want to change the button text from "Heatmaps" to "Add Heatmap Tracks"

Good catch. Fixed.

**Comment by alisman** [comment]
@Luke-Sikina just tested this and works great.  just one more thing:  see the missing space to the right of the clinical tracks bubble?

**Pull request merged**